applications/ParticleMechanicsApplication/custom_utilities/mpm_search_element_utility.h

```diff
@@ -47,8 +47,11 @@ namespace MPMSearchElementUtility
     void SearchElement(ModelPart& rBackgroundGridModelPart, ModelPart& rMPMModelPart, const std::size_t MaxNumberOfResults,
         const double Tolerance)
     {
+        ProcessInfo& rProcessInfo = rBackgroundGridModelPart.GetProcessInfo();
+        bool isExplicit = rProcessInfo.Has(IS_EXPLICIT);
```

Comment:
```suggestion
        const ProcessInfo& r_process_info = rBackgroundGridModelPart.GetProcessInfo();
        const bool is_explicit = rProcessInfo[IS_EXPLICIT];
```

I think you should check the value of `IS_EXPLICIT` and not only if it exists or not!
Remember to do this with a `const` instance of `ProcessInfo` otherwise you might secretly allocate `IS_EXPLICIT`, like [I explainer here](https://github.com/KratosMultiphysics/Kratos/pull/6691#issuecomment-609432609) (same applies to the `ProcessInfo`)

Reply:
Aye aye captain!